projects/storefrontstyles/scss/components/myaccount/order-history/_replenishment-order-history.scss

```diff
@@ -0,0 +1,200 @@
+%cx-replenishment-order-history {
+  .cx-replenishment-order-history {
+    @include media-breakpoint-down(sm) {
+      max-width: 100%;
+      padding: 0;
+    }
+  }
+  .cx-replenishment-order-history-table {
+    padding: 1.5rem 0 1.125rem 0;
+    margin-bottom: 0;
+    border-width: 1px 0 1px 0;
+    border-color: var(--cx-color-light);
+    border-style: solid;
+
+    tr {
+      width: 100%;
+
+      @include media-breakpoint-down(sm) {
+        border-width: 1px 0 0 0;
+        border-color: var(--cx-color-light);
+        border-style: solid;
+
+        &:first-child {
+          border-width: 0;
+          padding: 1.25rem 0 0 0;
+        }
+      }
+    }
+
+    th {
+      padding: 1.5rem 0 1.125rem 0;
+      padding-top: 1.5rem;
+      padding-inline-end: 0;
+      padding-bottom: 1.125rem;
+      padding-inline-start: 5px;
+      text-align: left;
+
+      &:first-child {
+        padding-inline-start: 0;
+      }
+
+      &:last-child {
+        text-align: right;
+      }
+    }
+
+    td {
+      width: 14.2%;
+      padding: 1.625rem 5px;
+
+      @include media-breakpoint-up(md) {
+        text-align: left;
+        &:first-child {
+          padding-inline-start: 0px;
+        }
+
+        &:last-child {
+          text-align: right;
+        }
+      }
+
+      @include media-breakpoint-down(sm) {
+        width: 100%;
+        display: flex;
+        border-width: 0;
+        padding: 0 1.25rem;
+
+        &:first-child {
+          padding-top: 1.25rem;
+        }
+
+        &:last-child {
+          padding-bottom: 1.25rem;
+          padding-top: 7px;
+        }
+      }
+    }
+  }
+
+  .cx-replenishment-order-history-header {
+    padding: 40px 0 0 0;
+    color: var(--cx-color-text);
+
+    @include media-breakpoint-down(sm) {
+      padding: 40px 20px 0 20px;
+    }
+  }
+
+  .cx-replenishment-order-history-thead-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-replenishment-order-history-code {
+    .cx-replenishment-order-history-value {
+      text-decoration: underline;
+    }
+  }
+
+  .cx-replenishment-order-history-placed {
```

Comment:
*-placed, and *-status can be group as they do the same in the end.

.cx-replenishment-order-history-placed,
.cx-replenishment-order-history-status {
etc
}

Reply:
these classes are removed as they were not doing anything special